src/index.js

```diff
@@ -42,15 +42,18 @@ function normalizeOptions(file) {
     if (typeof opts.root === 'string') {
       opts.root = [opts.root];
     }
-    opts.root = opts.root.reduce((resolvedDirs, dirPath) => {
-      if (glob.hasMagic(dirPath)) {
-        return resolvedDirs.concat(
-          glob.sync(dirPath)
-            .filter(resolvedPath => fs.lstatSync(resolvedPath).isDirectory()),
-        );
-      }
-      return resolvedDirs.concat(dirPath);
-    }, []);
+    opts.root = opts.root
+      .map(dirPath => path.resolve(opts.cwd, dirPath))
+      .reduce((resolvedDirs, absDirPath) => {
+        if (glob.hasMagic(absDirPath)) {
+          const roots = glob.sync(absDirPath)
+            .filter(resolvedPath => fs.lstatSync(resolvedPath).isDirectory());
+
+          return [...resolvedDirs, ...roots];
+        }
+
+        return [...resolvedDirs, absDirPath];
```

Comment:
I decided to switch to explicit array spread instead of `concat`, as I'm not a fan of its dual behavior (either push-like or actual concatenation).